iis_action.ps1

```diff
@@ -0,0 +1,71 @@
+Param(
+    [parameter(Mandatory = $true)]
+    [ValidateSet( 'start', 'stop', 'restart')]
```

Comment:
Is this case sensitive?  It's probably worth a note on the readme because your script assumes it's lowercase.